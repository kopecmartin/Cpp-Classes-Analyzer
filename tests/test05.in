class A {
    virtual int foo(void) =0;
};

class B {};

class C : A {};

class D : B, C {};