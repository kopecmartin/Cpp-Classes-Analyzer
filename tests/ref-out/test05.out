<?xml version="1.0" encoding="UTF-8"?>
<model>
    <class name="A" kind="abstract">
        <class name="C" kind="abstract">
            <class name="D" kind="abstract" />
        </class>
    </class>
    <class name="B" kind="concrete">
        <class name="D" kind="abstract" />
    </class>
</model>
